templates/edit.html

```diff
@@ -13,7 +13,7 @@
                 <h1>{{ object.name.get_value }}</h1>
                 <p>
                     {% trans "ID:" %} {{ object.uuid }}<br />
-                    {% trans "Last update by:" %} username goes here <br />
+                    {% trans "Last update by:" %} {% trans "username goes here" %} <br />
```

Comment:
I'm assuming we're going to update these strings to show actual data once we revisit versioning. For now, however, since we're showing a dummy string, it should probably be translated, too.